Terminal.Gui/View/Layout/Dim.cs

```diff
@@ -7,7 +7,7 @@ namespace Terminal.Gui;
 ///     <para>
 ///         A Dim object describes the dimensions of a <see cref="View"/>. Dim is the type of the
 ///         <see cref="View.Width"/> and <see cref="View.Height"/> properties of <see cref="View"/>. Dim objects enable
-///         Computed Layout (see <see cref="LayoutStyle.Computed"/>) to automatically manage the dimensions of a view.
+///         automatic management of the dimensions of a view.
```

Comment:
I kinda think that whole sentence should just go, since it's almost a repetition of the previous sentence.